front/app/containers/Admin/pagesAndMenu/EditHomepage/BottomInfoSection/index.tsx

```diff
@@ -0,0 +1,34 @@
+import useHomepageSettings from 'hooks/useHomepageSettings';
+import React, { lazy } from 'react';
+import { InjectedIntlProps } from 'react-intl';
+import { updateHomepageSettings } from 'services/homepageSettings';
+import { injectIntl } from 'utils/cl-intl';
+import { isNilOrError } from 'utils/helperUtils';
+import { homeBreadcrumb } from '../../breadcrumbs';
+
+const GenericBottomInfoSection = lazy(
```

Comment:
I don't think this needs to be `lazy` since as part of this component, it will always render